test.sh

```
@@ -0,0 +1,45 @@
+# ****************************************************************************
+# Get all the setup.sh files and execute them to check if any errors occure
+# Created by: 
+# ****************************************************************************
+
+cd basic-commits
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)